### ISSUE TITLE
Battle Equipment

### ISSUE THREAD

**Issue by Dohtur-Frag** [Kurzdor/wotmods-public#7]
Не совсем бага, но всё же.
До того как появилась возможность двигать окно по экрану - всё было хорошо.
Теперь, иконка "свернуть/развернуть окно" отображается поверх иконки первого снаряда (при левом отображение окна).
Можно ли как-то решить эту проблему?
Например сдвинуть окно левее или сделать иконку "умной" - выводить её с противоположной стороны от ближайшего объекта, а ещё лучше выводить её внутри окна - это было бы универсально.

**Comment by Kurzdor** [comment]
@Dohtur-Frag Не вижу в этом проблемы. Ничего не мешает нажать её там, где она сейчас. Функционал не сломан. Я её разве что чуть-чуть подвину на расстояние, которое есть сейчас между слотами.

**Comment by Dohtur-Frag** [comment]
Проблема не в нажать, а в том, что если не знать, что она есть, её легко не заметить.
Подозреваю, что если стандартная панель будет заменена модом (видел такой, меняет картинки снарядов), то можно совсем не разглядеть из-за более другого фона.

**Comment by Kurzdor** [comment]
@Dohtur-Frag не спорю, но тогда будет некрасиво большой отступ, который был раньше. Проблему с нестандартной панелью я решил частично драгом и может быть ещё подумаю как подружить вместе с ней.

**Comment by Dohtur-Frag** [comment]
Ещё один момент.
Переключать снаряды можно мышкой. Возможно таких игроков не много, но это создаёт реальную проблему.
При выборе мышкой, снаряда в первом слоте, очень часто попадаешь в эту самую кнопку "свернуть/развернуть".

**Comment by Kurzdor** [comment]
Частично исправлено в 3.0.2.
Пока что не закрываю, жду фидбека.

**Comment by Dohtur-Frag** [comment]
3.0.2
Если не считать немного изменившегося расстояния между иконками, остальное осталось так же, т.е. "свернуть" лежит на иконке первого снаряда.

ПС
Слева от иконок, отображается номер выбранного набора.
Там же есть место. Может "свернуть" перенести туда, чтоб она стала частью панели?

**Comment by Kurzdor** [comment]
@Dohtur-Frag нет, минимайзер должен быть справа железобетонно.

**Comment by Dohtur-Frag** [comment]
Ок.
Тогда можно перенести номер на право и появиться место для "свернуть".

**Comment by Dohtur-Frag** [comment]
Как вы смотрите на то, чтобы отказаться от кнопки "свернуть" и сделать хоткей "вкл/выкл" отображение?

**Comment by Kurzdor** [comment]
@Dohtur-Frag крайне негативно. Юзеру сложней помнить "какой же там хоткей у меня стоял или стоит" для такой-то функции.

**Comment by Dohtur-Frag** [comment]
Сложно добавить хинты при наведении на оборудование?

**Comment by Kurzdor** [comment]
@Dohtur-Frag имеешь ввиду всплывающие окна с информацией на слотах как в ангаре? Хочу, надо будет поресёрчить данный вопрос.

**Comment by GL00MNA** [comment]
This is a great MOD, I hope you can continue updating.
Thanks from Canada.

**Comment by Kurzdor** [comment]
@GL00MNA thanks a lot :)

**Comment by Dagovax** [comment]
I can't move the panel around. I tried multiple combo's: Control + Left Mouse. Right mouse, + alt + shift idk what but it won't move...

**Comment by Kurzdor** [comment]
@Dagovax did you enable Drag option in settings, i.e. free positioning?

**Comment by Dagovax** [comment]
I am using Aslain's modpack. During installation I tried both options (left/right). Both not enabling the option to move

**Comment by Kurzdor** [comment]
@Dagovax this mod has Mods Settings API support. Open settings in hangar and find Battle Equipment section. There you can find setting "Position" and option "Free" which enabled dragging and unlocks panel from consumables panel, it doesn't respect it's position and sizes unlike in "Left" or "Right" options.

See the screens:

<details>
<summary>Screenshots</summary>
<img src="https://github.com/user-attachments/assets/f3798e6a-9df4-4a13-975b-5a49dc9e4a88" />
<img src="https://github.com/user-attachments/assets/c7b65e23-9ec5-409d-a98c-08f89701210d" />
</details>

**Comment by Todeskult** [comment]
The newest file for 1.28.0.0 cannot be downloaded. MS Edge says "Couldn't download - Virus detected".

![Image](https://github.com/user-attachments/assets/0a6f113d-a249-43c2-828d-000331936c54)

**Comment by Kurzdor** [comment]
> The newest file for 1.28.0.0 cannot be downloaded. MS Edge says "Couldn't download - Virus detected".
> 
> ![Image](https://github.com/user-attachments/assets/0a6f113d-a249-43c2-828d-000331936c54)

https://www.virustotal.com/gui/file/680ad83d8e74daf0a998afdac8bd5350d31b4007822c231396e30e65ab085c93?nocache=1
False positive from Windows Defender. Sometimes it can be too aggressive on simple files.

**Comment by Todeskult** [comment]
> > The newest file for 1.28.0.0 cannot be downloaded. MS Edge says "Couldn't download - Virus detected".
> > ![Image](https://github.com/user-attachments/assets/0a6f113d-a249-43c2-828d-000331936c54)
> 
> https://www.virustotal.com/gui/file/680ad83d8e74daf0a998afdac8bd5350d31b4007822c231396e30e65ab085c93?nocache=1 False positive from Windows Defender. Sometimes it can be too aggressive on simple files.

Yeah, there's something with not the Edge itself but with Windows Defender, not allowing this file into my system. I've tried multiple ways of putting this file: downloaded with no issues on different PC, then shared in on Google Drive, etc. But my Windows just doesn't like this file and deletes it.

**Comment by Todeskult** [comment]
I've tried to also mail the ZIP file to my mailbox, but this failed too, my provider's server also claims there's a virus inside and refuses even to accept the mail, it got bounced back.

So I made another workaround that helped. So if there's anybody else with similar problem here's the solution:
1. I have downloaded the file on a different PC without issues (used Firefox browser to be sure)
2. Unpacked the ZIP
3. Then I have copied the **me.kurzdor.battleequipment_3.4.3.wotmod** only to my Google Drive webpage (no issues there with a virus being detected). I didn't tried to copy the other wotmod or config json files, as I have them already from different mods that I'm using and the previous version of the mod (json file).
4. Got the file on my destination PC, however not via the Edge browser, but with the help of Google Drive desktop app, moving **me.kurzdor.battleequipment_3.4.3.wotmod** file from my local G drive. Voilla!
